website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -18,8 +18,14 @@ upgrade flow.
 
 ### Service Mesh Compatibility
 
-Existing Consul deployments using service mesh (i.e., containing any registered 
-Connect proxies) should upgrade to **at least Consul 1.13.1**.
+Existing Consul deployments using service mesh
+(i.e., containing any registered Connect proxies)
+must consider the following before upgrading to Consul 1.13.x
+depending on their configuration.
```

Comment:
Thanks - I'll try to improve.

The three subsections below are meant to describe the three conditions. Maybe I can just enumerate the subsections in a list (as a summary), since those are the three configurations that need to be considered.